applejack/conf/products/central-configuration.yml

```diff
@@ -76,3 +77,9 @@ context:
       value: "false"
     - key: CENTRAL_CONFIG_ENCRYPT_KEY
       value: ""
+    - key: USE_CACHE
+      value: "false"
+    - key: PROVIDER
```

Comment:
@Karthigaiselvan - can you check the story to see if these variables are required?